spacy/tokens/span.pyx

```diff
@@ -802,28 +802,18 @@ cdef class Span:
 
     property id:
         def __get__(self):
-            cdef SpanC* span_c = self.span_c()
-            return span_c.id
+            return self.span_c().id
 
         def __set__(self, attr_t id):
-            cdef SpanC* span_c = self.span_c()
-            span_c.id = id
+            self.span_c().id = id
 
     property ent_id:
```

Comment:
Should we mark `ent_id` as deprecated and refer to `span.id` instead so we can remove the former at some point in the future?

Reply:
It's used in so much code that I am not sure it makes sense to deprecate it when it's not really a problem to maintain this.

Reply:
It feels pretty weird to have both of these around forever, but you're right that it's not hard to maintain right now and users would expect both of them. It's not great but it's probably better to have both of them for now.